front-end/components/SettingsPanel/panels/Notify/notify.vue

```diff
@@ -14,12 +14,23 @@
         <button class="option" @click="enableNotifications" :class="{ current: pushNotifications }">
             Enable desktop notifications
         </button>
+        <ul id="statusSelector">
```

Comment:
what about adding a `v-if="pushNotifications"`? That way the other options are only shown when the notifications are enabled :)